packages/core/core/src/Parcel.js

```diff
@@ -361,6 +358,7 @@ export default class Parcel {
         createValidationRequest({optionsRef: this.#optionsRef, assetRequests}),
         {force: assetRequests.length > 0},
       );
+      await this.#requestTracker.writeToCache();
```

Comment:
This is after the build has reported as complete. Open to suggestions for a more suitable location.